qiskit/circuit/library/standard_gates/sx.py

```diff
@@ -178,7 +185,15 @@ def _define(self):
         self.definition = qc
 
     def inverse(self, annotated: bool = False):
-        """Return inverse SXdg gate (i.e. SX)."""
+        """Return inverse SXdg gate (i.e. SX).
+
+        Args:
+            annotated: indicates whether the inverse gate can be implemented
+                as an annotated gate.
```

Comment:
```suggestion
            annotated: Typically when set to ``True`` this is used to return an :class:`.AnnotatedOperation` 
                with an inverse modifier set instead of a concrete :class:`.Gate`. However, for this class this
                argument is ignored as this inverse of this gate is always a :class:`.SXGate`.
```